cloud/kubernetes/helm/yugabyte/values.yaml

```diff
@@ -4,13 +4,13 @@
 Component: "yugabytedb"
 Image:
   repository: "yugabytedb/yugabyte"
-  tag: latest
+  tag: 1.0.4.0-b24 
   pullPolicy: IfNotPresent
 
 persistentVolume:
   count: 2
   storage: 100Mi
```

Comment:
Is this the default value? Seems too low. We should make this at least 10GiB, otherwise its not a practical production value.

Also, this value can be overridden by the helm chart right?

Reply:
sure, will update to 10Gi, yeah this is overwritten from YugaWare